src/Workspaces/Core/Portable/Formatting/Formatter.cs

```diff
@@ -158,7 +159,13 @@ internal static async Task<Document> FormatAsync(Document document, SyntaxAnnota
         /// <param name="cancellationToken">An optional cancellation token.</param>
         /// <returns>The formatted tree's root node.</returns>
         public static SyntaxNode Format(SyntaxNode node, SyntaxAnnotation annotation, Workspace workspace, OptionSet options = null, CancellationToken cancellationToken = default)
-            => FormatAsync(node, annotation, workspace, options, cancellationToken).WaitAndGetResult_CanCallOnBackground(cancellationToken);
+        {
+            options = options ?? workspace.Options;
+            options = options.WithChangedOption(FormattingOptions.AllowConcurrent, false);
+            var resultTask = FormatAsync(node, annotation, workspace, options, cancellationToken);
+            Debug.Assert(resultTask.IsCompleted);
+            return resultTask.WaitAndGetResult_CanCallOnBackground(cancellationToken);
+        }
```

Comment:
why did these change?

Reply:
One of them is a required change; the others were updated for consistent behavior.